src/utils/object.ts

```diff
@@ -53,3 +53,23 @@ export function startsWith(target: string, searchString: string, position?: numb
 
   return target.indexOf(searchString, effectivePosition) === effectivePosition;
 }
+
+/**
+ * Returns true if the searchString sequence is the same as the corresponding target sequence
+ * starting at the given target position (defaulting to 0); otherwise returns false.
+ * @param target The string to search within.
+ * @param searchString The string to search for.
+ * @param position The position to start searching witin the target.
```

Comment:
I wonder if we should update the documentation for this `@param` to closer align with ES2015 `endsWith` params. The ES2015 `endsWith` function accepts an end position, whereas here we're describing this param as a start position (possibly a copy/paste leftover from the our local `startsWith` function). Could use MDN docs for inspiration: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

Reply:
updated the params to accurely reflect the usage of the params!